tests/event_lists/test_south_african.py

```
@@ -0,0 +1,405 @@
+    "Iran–South Africa relations": "العلاقات الإيرانية الجنوب الإفريقية",
+    "Iraq–South Africa relations": "العلاقات الجنوب الإفريقية العراقية",
+    "Israel–South Africa relations": "العلاقات الإسرائيلية الجنوب الإفريقية",
+    "Italy–South Africa relations": "العلاقات الإيطالية الجنوب الإفريقية",
+    "Italy–South Africa sports relations": "العلاقات الإيطالية الجنوب الإفريقية الرياضية",
+    "Ivory Coast–South Africa relations": "العلاقات الإيفوارية الجنوب الإفريقية",
+    "Jamaica–South Africa relations": "العلاقات الجامايكية الجنوب الإفريقية",
+    "Japan–South Africa relations": "العلاقات الجنوب الإفريقية اليابانية",
+    "Jordan–South Africa relations": "العلاقات الأردنية الجنوب الإفريقية",
+    "Kazakhstan–South Africa relations": "العلاقات الجنوب الإفريقية الكازاخستانية",
+    "Kenya–South Africa relations": "العلاقات الجنوب الإفريقية الكينية",
+    "Kenya–South Africa sports relations": "العلاقات الجنوب الإفريقية الكينية الرياضية",
```

Ruff (0.14.10)

Dictionary key literal `"Kenya–South Africa sports relations"` repeated

Remove repeated key literal `"Kenya–South Africa sports relations"`

(F601)